src/lib/containers/download_list_v2/DownloadCartPage.tsx

```diff
@@ -82,8 +82,10 @@ export const DownloadCartPage:React.FunctionComponent<DownloadCartPageProps> = (
               Clear Your Download Cart
             </a>
           </div>
-          <p className="description">You may find your added files in the tabs below. Any files which require actions before download can be found in the Access Actions Required tab,
```

Comment:
If `description` corresponds to a specific CSS class here, you may be able to delete it.

Reply:
Added `description` back!